packages/block-library/src/video/tracks.js

```diff
@@ -0,0 +1,5 @@
+export default function Tracks( { tracks = [] } ) {
+	return tracks.map( ( track ) => {
+		return <track key={ track.src } { ...track } />;
+	} );
```

Comment:
Maybe we should use a different variable name in the loop that can't be confused with the `<track />` tag name easily?

```suggestion
	return tracks.map( ( t ) => {
		return <track key={ t.src } { ...t } />;
	} );
```